debian/control

```diff
@@ -107,6 +107,7 @@ Build-Depends: automake,
                python3-venv,
                python3-werkzeug <pkg.ceph.check>,
                python3-yaml,
+               python3-packaging,
```

Comment:
python3-packaing is not a build dependency. it is a runtime dependency.

Reply:
and the commit message does not match with the change.

Reply:
@tchaikov Fixed.